admin_manual/configuration_server/occ_command.rst

```diff
@@ -1287,6 +1288,40 @@ authentication servers such as LDAP::
  | user directories | 2  |
  +------------------+----+
  
+Syncing User Accounts
+^^^^^^^^^^^^^^^^^^^^^
+
+This command syncs users stored in external backend services, such as _LDAP_, _Shibboleth_, and _Samba_, with ownCloud’s internal user database.
+
+.. note:: 
+   It’s also `one of the commands`_ that you should run on a regular basis to ensure that your ownCloud installation is running optimally.
+
+It’s not essential to run it regularly, unless you have a large number of users who change on a semi-regular basis.
+What’s more, the job is only necessary to pick up changes from a backend with users who don't log in regularly.
```

Comment:
@DeepDiver1975 is this correct ? Does the login command automatically sync other properties ?

The way I understood is that only the first login creates the user with properties, but subsequent logins don't.
Subsequent logins would also be expensive especially with clients that don't support cookies/sessions.